src/RestControllers/AuthorizationController.php

```diff
@@ -864,6 +955,7 @@ private function deserializeUserSession(): AuthorizationRequest
             $authRequest->setCodeChallenge($outer['codeChallenge']);
             $authRequest->setCodeChallengeMethod($outer['codeChallengeMethod']);
         } catch (Exception $e) {
+            // TODO: @adunsulag check with @sjpadgett was just echoing the exception what you wanted to happen here?
```

Comment:
@sjpadgett question for you here, in AuthorizationController I see we are just echoing out the exception, is this what you wanted as part of the oauth2 flow?